test/membrane_rtc_engine/tee_test.exs

```diff
@@ -239,8 +244,6 @@ defmodule Membrane.RTC.Engine.TeeTest do
     mark_variant_as_resumed(pipeline, :high)
     assert_sink_event(pipeline, :sink, %TrackVariantResumed{variant: :high})
 
-    activate_source(pipeline, :high)
```

Comment:
we should send some buffers here to make sure endpoint has to re-request track after being inactive and active again